internal/sign/ocpbuild/maker.go

```diff
@@ -0,0 +1,255 @@
+		CommonSpec: buildv1.CommonSpec{
+			ServiceAccount: constants.OCPBuilderServiceAccountName,
+			Source:         sourceConfig,
+			Strategy: buildv1.BuildStrategy{
+				Type: buildv1.DockerBuildStrategyType,
+				DockerStrategy: &buildv1.DockerBuildStrategy{
+					Volumes: []buildv1.BuildVolume{
+						{
+							Name: "key",
+							Source: buildv1.BuildVolumeSource{
+								Type: buildv1.BuildVolumeSourceTypeSecret,
+								Secret: &v1.SecretVolumeSource{
+									SecretName: signConfig.KeySecret.Name,
+									Optional:   pointer.Bool(false),
+								},
+							},
+							Mounts: []buildv1.BuildVolumeMount{
+								{DestinationPath: "/run/secrets/key"},
+							},
+						},
+						{
+							Name: "cert",
+							Source: buildv1.BuildVolumeSource{
+								Type: buildv1.BuildVolumeSourceTypeSecret,
+								Secret: &v1.SecretVolumeSource{
+									SecretName: signConfig.CertSecret.Name,
+									Optional:   pointer.Bool(false),
+								},
+							},
+							Mounts: []buildv1.BuildVolumeMount{
+								{DestinationPath: "/run/secrets/cert"},
+							},
+						},
+					},
+				},
+			},
+			Output:         buildTarget,
+			NodeSelector:   mld.Selector,
+			MountTrustedCA: pointer.Bool(true),
+		},
+	}
+
+	hash, err := m.hash(ctx, &spec, mld.Namespace, signConfig.KeySecret.Name, signConfig.CertSecret.Name)
+	if err != nil {
+		return nil, fmt.Errorf("could not hash Build spec: %v", err)
+	}
+
+	build := &buildv1.Build{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: mld.Name + "-sign-",
+			Namespace:    mld.Namespace,
+			Labels:       buildutils.GetBuildLabels(mld, BuildType),
+			Annotations:  buildutils.GetBuildAnnotations(hash),
+		},
+		Spec: buildv1.BuildSpec{
+			CommonSpec: buildv1.CommonSpec{
+				ServiceAccount: constants.OCPBuilderServiceAccountName,
+				Source:         sourceConfig,
+				Strategy: buildv1.BuildStrategy{
+					Type: buildv1.DockerBuildStrategyType,
+					DockerStrategy: &buildv1.DockerBuildStrategy{
+						Volumes: []buildv1.BuildVolume{
+							{
+								Name: "key",
+								Source: buildv1.BuildVolumeSource{
+									Type: buildv1.BuildVolumeSourceTypeSecret,
+									Secret: &v1.SecretVolumeSource{
+										SecretName: signConfig.KeySecret.Name,
+										Optional:   pointer.Bool(false),
+									},
+								},
+								Mounts: []buildv1.BuildVolumeMount{
+									{DestinationPath: "/run/secrets/key"},
+								},
+							},
+							{
+								Name: "cert",
+								Source: buildv1.BuildVolumeSource{
+									Type: buildv1.BuildVolumeSourceTypeSecret,
+									Secret: &v1.SecretVolumeSource{
+										SecretName: signConfig.CertSecret.Name,
+										Optional:   pointer.Bool(false),
+									},
+								},
+								Mounts: []buildv1.BuildVolumeMount{
+									{DestinationPath: "/run/secrets/cert"},
+								},
+							},
+						},
+					},
+				},
+				Output:         buildTarget,
+				NodeSelector:   mld.Selector,
+				MountTrustedCA: pointer.Bool(true),
+			},
+		},
```

Comment:
```suggestion
		Spec: spec,
```

Reply:
Wow, not sure how I missed this duplication. Thanks!